.gitignore

```diff
@@ -123,7 +123,5 @@ examples/interactive_asr/data/*.model
 examples/interactive_asr/data/*.pt
 
 # third parties
-third_party/archives/
 third_party/install/
-third_party/src/
-third_party/tmp/
```

Comment:
nit: the folders `third_party/sox/src` and `third_party/sox/tmp` will no longer be ignored. is that intentional?

Reply:
yes, they will go to build directory.